src/models/Settings.js

```diff
@@ -472,4 +472,8 @@ export default Model.extend({
 
     return options;
   },
+
+  isDsTheme: function () {
```

Comment:
FYI:
BaseForm expects the `this.settings` is instance of `Courage/SettingsModel` but apparently SIW creates its own settings (`src/models/Settings`) which is incompatible with Courage's SettingsModel.